website/notifications/emails.py

```diff
@@ -0,0 +1,170 @@
+import pytz
```

Comment:
NP: Imports should be grouped into system, third-party, and local source tree each of which are separated by a single empty line.